wandb/integration/sb3/sb3.py

```diff
@@ -86,7 +86,7 @@ def __init__(
         model_save_path: Optional[str] = None,
         model_save_freq: int = 0,
         gradient_save_freq: int = 0,
-        gradient_log: Literal["gradients", "parameters", "all"] = "all",
+        log: Literal["gradients", "parameters", "all"] = "all",
```

Comment:
Add the no log option? This watch is really slows down training so we should give people option to turn it off